### PR TITLE
[ISSUE #38]support payload with byte[] type in message conversion

### PR DESCRIPTION
**Pull request by fangjian0423**
ISSUE #38 
Support payload with byte[] type in message conversion.

### CODE REVIEW

**Comment by walking98** [comment]
LGTM

**Comment by vongosling** [comment]
@fangjian0423 Could you add the est for byte array processing？

File: rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/support/RocketMQUtil.java [hunk 1]
```diff
@@ -114,6 +114,8 @@ public static MessagingException convert(MQClientException e) {
 
         if (payloadObj instanceof String) {
             payloads = ((String) payloadObj).getBytes(Charset.forName(charset));
+        } else if (message.getPayload() instanceof byte[]) {
```
Comment (ShannonDing) [inline rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/support/RocketMQUtil.java]: It is better, IMO, to keep the code style same as line 115.

Comment (ShannonDing) [reply rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/support/RocketMQUtil.java]: payloadObj instanceof byte[] maybe better.

Comment (fangjian0423) [reply rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/support/RocketMQUtil.java]: done

**Comment by fangjian0423** [comment]
> @fangjian0423 Could you add the est for byte array processing？

I write some test case for `RocketMQUtil#convertToRocketMessage`.

**Pull request merged**